FSJ_django20_project/FSJ/models_program.py

```diff
@@ -0,0 +1,13 @@
+import uuid
+from django.utils.translation import gettext_lazy as _
+from django.db import models
+
+class Program(models.Model):
+    class Meta():
+        ordering = ('name',)
+        
+    programid = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
```

Comment:
If I remember correctly their programs already have IDs which were included in the data they sent us. Might make a bit more sense to have those as the primary key and make them an editable field than to create our own ID field.